test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Common/IProjectContextInfoExtensionsTests.cs

```diff
@@ -93,12 +94,167 @@ public async Task GetInstalledPackagesAsync_WhenProjectContextInfoIsNull_Throws(
         {
             await VerifyMicrosoftAssumesExceptionAsync(
                 () => IProjectContextInfoExtensions.GetInstalledPackagesAsync(
-                    projectContextInfo: null,
+                    projectContextInfos: null,
                     Mock.Of<IServiceBroker>(),
                     CancellationToken.None)
                 .AsTask());
         }
 
+        [Fact]
+        public async Task GetInstalledPackagesAsync_WhenProjectContextInfoEmpty_DoesNotInvokeService()
+        {
+            // Arrange
+            var serviceBroker = new Mock<IServiceBroker>();
+            var projectManagerService = new Mock<INuGetProjectManagerService>();
+            var dictionary = new Dictionary<string, IReadOnlyCollection<IPackageReferenceContextInfo>>();
+            var expectedResult = new ReadOnlyDictionary<string, IReadOnlyCollection<IPackageReferenceContextInfo>>(dictionary);
```

Comment:
You can delete this statement and just use `dictionary` directly.  `Dictionary<TKey, UValue>` already implements `IReadOnlyDictionary<TKey, UValue>`.